README.md

```diff
@@ -26,10 +27,10 @@ If you need to edit the configuration, you can publish it with:
 php artisan vendor:publish --provider="Sopamo\LaravelFilepond\LaravelFilepondServiceProvider"
 ```
 
-When you receive the serverId from Filepond (that's the value which you get via the hidden input fields) in your controller you can decode it via:
+Included in this repo is a Filepond upload controller which is where you should direct uploads to. Upon upload the controller will return the `$serverId` which can be used in your own controller to move the file from temporary storage to somewhere permanent using the `getPathFromServerId()` function.
```

Comment:
I think we should not here, that the serverId is usually handed over via a hidden input field. As far as I know that's done by filepond by default.